chain/network/src/peer_manager/testonly.rs

```diff
@@ -382,6 +390,14 @@ impl ActorHandler {
                 .await;
         }
     }
+
+    pub async fn tier1_connect(&self, clock: &time::Clock) {
```

Comment:
add comment. How does it differ from other 'connect' methods?

Reply:
done